disk-utils/sfdisk.c

```diff
@@ -2306,6 +2318,9 @@ int main(int argc, char *argv[])
 		{ 'F','d'},                     /* --list-free --dump */
 		{ 'F','J'},                     /* --list-free --json */
 		{ 's','u'},			/* --show-size --unit */
+		{ OPT_NO_DEVICE_NAMES, 'l' },   /* --no-device-names --list (meaningless) */
+		{ OPT_NO_DEVICE_NAMES, 'F' },   /* --no-device-names --list-free (meaningless) */
+		{ OPT_NO_DEVICE_NAMES, 'V' },   /* --no-device-names --verify (meaningless) */
 		{ 0 }
```

Comment:
To make the option collision check work, you will have to align the options in ascending ASCII order as well as the option groups themselves. That is:
```suggestion
        { 'F','d'},                     /* --list-free --dump */
		{ 'F','J'},                     /* --list-free --json */
		{ 'F', OPT_NO_DEVICE_NAMES },   /* --no-device-names --list-free (meaningless) */
		{ 'V', OPT_NO_DEVICE_NAMES },   /* --no-device-names --verify (meaningless) */
		{ 'l', OPT_NO_DEVICE_NAMES },   /* --no-device-names --list (meaningless) */
		{ 's','u'},			/* --show-size --unit */
```